### PR TITLE
Move findColumnBy{ID,Name} to methods on TableDescriptor.

### PR DESCRIPTION
**Pull request by petermattis**

### CODE REVIEW

**Comment by vivekmenezes** [comment]
LGTM

File: structured/structured.go [hunk 1]
```diff
@@ -188,6 +188,28 @@ func (desc *TableDescriptor) Validate() error {
...
+		if c.Name == name {
+			return c, nil
+		}
+	}
+	c := ColumnDescriptor{}
+	return c, fmt.Errorf("column \"%s\" does not exist", name)
+}
+
+// FindColumnByID finds the column with specified ID.
+func (desc *TableDescriptor) FindColumnByID(id uint32) (ColumnDescriptor, error) {
```
Comment (tamird) [inline structured/structured.go]: shouldn't this return a pointer so that it can be nil in the error case?

Comment (petermattis) [inline structured/structured.go]: I don't think it matters, but I've changed it.

**Comment by tamird** [comment]
LGTM

**Pull request merged**